### PR TITLE
Unity studies filter

### PR DESCRIPTION
**Pull request by atatmaja**
## Briefly describe the feature or bug that this PR addresses.

- Adds unity studies checkbox and /Unity view (same as explore but with checkbox checked off by default)
- Changes filters and filter options data models to use arrays instead of sets (sets were an artifact from when we were doing filtering logic on the frontend)
- Removes some unnecessary code (dataPageState, some types that were not being used, API call that's not being used, unnecessary dispatches)

## Please link the Airtable ticket associated with this PR.

https://airtable.com/tbli2lWQHAqBa6ZcI/viwWvxb27XGuziw4c/recGnymO05NXQZDqX?blocks=bipIkdtYHmboUfJut

## If applicable, include screenshots of the feature/bugfix introduced by this PR (Please include both desktop and mobile if there is a significant UI change).

image
image
image
image

## Describe the steps you took to test the feature/bugfix introduced by this PR.

Smoke test

## Does this PR depend on any recent backend work? If so, please include the PR number from the iit-backend repo and associated Airtable ticket link.

Yes, but PR has been merged

## Does this PR require any French translations? Have they been included?

Yes, will include them all once we figure out exactly what text we want to show

### CODE REVIEW

File: src/components/shared/InformationIcon.tsx [hunk 1]
```diff
@@ -23,6 +23,7 @@ export default function InformationIcon(props: InformationIconProps) {
         offset={offset}
         position={position}
         size={popupSize}
+        hoverable={true}
```
Comment (annoyingpr) [inline src/components/shared/InformationIcon.tsx:26]: nit: this should work too right?
```suggestion
        hoverable

```

File: src/utils/translate/en.json [hunk 2]
```diff
@@ -448,8 +449,17 @@
...
   "StudyInformation": "Study Information",
   "StudyInformationTooltip": {
-    "FirstParagraph": "Filter on study details, including source type (publication, preprint, news, or report) and study status (ongoing, completed).",
-    "SecondParagraph": "Risk of bias: Reflects the extent to which the true prevalence may be different from the estimated prevalence. Estimated by SeroTracker reviewers based on the Joanna Briggs Institute critical appraisal tool for prevalence estimates."
+    "FirstParagraph": "Filter on study details, including source type (publication, preprint, news, or report), study status (ongoing, completed), and alignment with the WHO Unity protocol.",
+    "SecondParagraph": {
+      "PartOne": "Risk of bias: ",
+      "PartTwo": "Reflects the extent to which the true prevalence may be different from the estimated prevalence. Estimated by SeroTracker reviewers based on the Joanna Briggs Institute critical appraisal tool for prevalence estimates."
+    },
+    "ThirdParagraph": {
```
Comment (annoyingpr) [inline src/utils/translate/en.json:457]: I know we're overhauling translation soon, but think we should not include extra spaces at the end of text in this JSON, that should be handled elsewhere.

File: src/components/sidebar/right-sidebar/Filters.tsx [hunk 3]
```diff
@@ -95,12 +83,26 @@ export default function Filters({ page }: FilterProps) {
...
             })
           }}
-          defaultValue={getFilters(filter_type)}
+          defaultValue={pageState.filters[filter_type] as string[]}
         />
       </div>
     )
   }
 
+  const buildFilterCheckbox = (filter_type: FilterType, label: string) => {
```
Comment (annoyingpr) [inline src/components/sidebar/right-sidebar/Filters.tsx:92]: Not sure how this worked before, but what is this used for & why was this added?

File: src/App.tsx [hunk 4]
```diff
@@ -156,6 +154,13 @@ function App() {
         <Route path="/:language/Insights">
           <Insights />
         </Route>
+        <Route path="/:language/Unity">
+          <Explore initialFilters={(() => {
+            const initialFilters = getEmptyFilters();
+            initialFilters.unity_aligned_only = true;
```
Comment (annoyingpr) [inline src/App.tsx:160]: Didn't know you could update `const` - but read more about it here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const. JavaScript is weird

**Review by annoyingpr — APPROVED** [review]
Just a few questions but LGTM 🚀

File: src/App.tsx (hunk 4 above)
Comment (atatmaja) [reply src/App.tsx:160]: yeah since we are not technically updating the reference we can update the const like this. this is a bit sketchy tho, I will change it to a `let` so what we're doing here is more apparent

File: src/components/sidebar/right-sidebar/Filters.tsx (hunk 3 above)
Comment (atatmaja) [reply src/components/sidebar/right-sidebar/Filters.tsx:92]: This is a new function that renders a checkbox filter rather than our typical dropdown filter. Needed to create this for the unity studies checkbox that was newly implemented
